package.json

```diff
@@ -4,8 +4,9 @@
   "description": "LogDNA Collector Agent",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint -c .eslintrc *.js **/*.js",
-    "unit": "./node_modules/.bin/jest"
+    "lint": "eslint .",
+    "pretest": "npm run lint",
```

Comment:
@darinspivey, why don't you just put `eslint .` instead of `npm run lint` since you have already removed that part from `.circleci/config.yml` as well?

Reply:
It's just to reuse the alias.  That way we're not duplicating commands, and `npm run lint` can work on its own.